build.toml

```diff
@@ -82,7 +82,16 @@ project = '{network_comm}'
 target = 'My Computer'
 build_spec = 'Move Source'
 
-[package]
+[[package]]
 type = 'nipkg'
+control_file = 'control'
 payload_dir = 'Built'
 install_destination = 'ni-paths-LV{labview_version}DIR'
+package_output_dir = 'Built'
```

Comment:
You don't need this line.

Reply:
I had just included it for readability. Removed now.